data_safe_haven/commands/deploy_sre_command.py

```diff
@@ -108,14 +118,16 @@ def handle(self) -> None:
             # Provision SRE with anything that could not be done in Pulumi
             manager = SREProvisioningManager(config, self.sre_name)
             manager.run()
+            return 0
 
         except DataSafeHavenException as exc:
             for (
                 line
-            ) in f"Could not deploy Secure Research Environment {self.argument('name')}.\n{str(exc)}".split(
+            ) in f"Could not deploy Secure Research Environment {self.sre_name}.\n{str(exc)}".split(
                 "\n"
             ):
                 self.error(line)
+        return 1
```

Comment:
Does this automatically get interpreted as an exit code?

Reply:
It's not too late to dump cleo and use one of the other CLI builders instead. IMO it tries to do to much (and be a bit too clever) which is why it's also got a load of IO and formatting functions in there too.

Reply:
Skeleton discussion [here](https://github.com/alan-turing-institute/data-safe-haven/discussions/1394)